views/error/index.tsx

```diff
@@ -34,20 +34,25 @@ const Error: FC<ErrorProps> = ({ message, linkGoTo }) => {
           </Motion>
           <DiedComputerSVG maxHeight="100%" maxWidth="100%" width="100%" />
         </Box>
-        <Typography variant="displayLarge" mb=".5rem" textAlign="center">
+        <Typography
+          mb=".5rem"
+          size="large"
+          variant="display"
+          textAlign="center"
+        >
           OOPS!
         </Typography>
         <Typography
           mb="1rem"
           textAlign="center"
-          variant="title2"
+          variant="headline"
+          size="large"
           fontFamily="Share Tech Mono"
```

Comment:
Do we have this fontFamily in our variants? If yes, you shouldn't add the property here.